.travis.yml

```diff
@@ -4,6 +4,8 @@ python:
 cache: pip
 install:
   - pip install -r requirements.txt
+before_script:
+  - black --check . || true

```

Comment:
This currently doesn't **fail** Travis CI build. `|| true` is added for that purpose. 
P.S.: Contributors need to know this introduction before seeing failures because of this.

If we want black to fail Travis builds, we need to replace the line with:
```yaml
before_script:
  - black --check .
```

Reply:
@chttrjeankr I think we'd have to move this to the one with the possibility of failures once we configure travis to build versions successfully with the API key.

Reply:
Yeah! Maintaining code quality would require `|| true` to go.